man/systemd.netservice.xml

```diff
@@ -0,0 +1,168 @@
+    take precedence over those in <filename>/usr/lib</filename>. Drop-in files under any of these
+    directories take precedence over the main network service file wherever located. (Of course, since
+    <filename>/run</filename> is temporary and <filename>/usr/lib</filename> is for vendors, it is
+    unlikely drop-ins should be used in either of those places.)</para>
+  </refsect1>
+
+  <refsect1>
+    <title>[Service] Section Options</title>
+
+      <para>The network service file contains a <literal>[Service]</literal>
+      section, which specifies a discoverable network service announced in a
+      local network with Multicast DNS broadcasts.</para>
+
+      <variablelist class='network-directives'>
+        <varlistentry>
+          <term><varname>Name=</varname></term>
+          <listitem>
+            <para>An instance name of the network service as defined in <ulink
+            url="https://tools.ietf.org/html/rfc6763">RFC 6763</ulink>, e.g. "webserver".
+            If the instance name contains a substring "%h" then the substring is
+            replaced with the current host name.</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term><varname>Type=</varname></term>
+          <listitem>
+            <para>A type of the network service, e.g. "_http._tcp".</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term><varname>Port=</varname></term>
+          <listitem>
+            <para>A port number of the network service.</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term><varname>Priority=</varname></term>
+          <listitem>
+            <para>A priority number set in SRV resource records corresponding to the network service.</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term><varname>Weight=</varname></term>
+          <listitem>
+            <para>A weight number set in SRV resource records corresponding to the network service.</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term><varname>Txt=</varname></term>
+          <listitem>
+            <para>A whitespace-separated list of arbitrary key/value pairs
+            conveying additional information about the named service, e.g. "path=/portal/index.html".</para>
+
+            <para>Since whitespaces are used to separate the pairs in the configuration file
+            they cannot be used in the values.</para>
+          </listitem>
+        </varlistentry>
+      </variablelist>
+
+  </refsect1>
+
+  <refsect1>
+    <title>Examples</title>
+    <example>
+      <title>FTP service</title>
+
+      <programlisting># /etc/systemd/resolve/ftp.netservice
+[Service]
+Name=%h
+Type=_ftp._tcp
+Port=21</programlisting>
+
+      <para>This makes the ftp server running on the host discoverable in the local network.</para>
```

Comment:
Could we maybe add a command here that shows how this can be discovered (using systemd-resolve or some other tool)?

Reply:
I'll add an example for `systemd-resolve` resolving a SRV record, but it doesn't fully support network browsing at the moment. Perhaps for now would be better to put here an example with `avahi-browse` running on a remote host, e.g. `avahi-browse -a -r`.